front-end/lib/widgets/ui/nav/controls/shared/server_manager_widget.dart

```diff
@@ -7,6 +7,10 @@ import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
 class ServerManagerWidget extends ConsumerWidget {
+  final double serverIconSize;
+
+  const ServerManagerWidget(this.serverIconSize);
```

Comment:
Could you change to `ServerManagerWidget({this.serverIconSize})` and adapt the usage of `ServerManagerWidget` like `ServerManagerWidget(serverIconSize: XXXX)`